### ISSUE TITLE
Codespace prebuild are no longer triggered

### ISSUE THREAD

**Issue by tobiasdiez** [sagemath/sage#36978]
### Steps To Reproduce

https://github.com/sagemath/sage/actions/workflows/codespaces/create_codespaces_prebuilds
(and noted by @kwankyu elsewhere)

### Expected Behavior

Have a prebuild for every (beta) release.

### Actual Behavior

Stopped building releases about two months ago.

### Additional Information

Probably the quota for the codespace billing has to be increased slightly. @sagemath/core could you please have a look?

(And also please replace my billing information if you have not done so already. Thanks!)

### Environment

```markdown
- **OS**:
- **Sage Version**:
```

### Checklist

- [X] I have searched the existing issues for a bug report that matches the one I want to file, without success.
- [X] I have read the documentation and troubleshoot guide

**Comment by williamstein** [comment]
You're right in that when I go to https://github.com/sagemath/sage/settings/codespaces it looks like this:

<img width="1006" alt="image" src="https://github.com/sagemath/sage/assets/1276278/27437c07-3e58-4fe8-9260-74285350151f">

There are some monthly donations to sagemath via "Open Source Collective", which would make sense to use for this.  I don't know how making that work automatically works at all.

I also have no idea how much codespace prebuilds cost.

**Comment by tobiasdiez** [comment]
Thanks for looking into this. You would need to add the bank or PayPal account under Billing information.

The prebuilds are cheap. I think I set the spending limit to 2 dollars a month, which is a bit too low but almost sufficient. Maybe 5$,max 10$? https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces#storage-costs-for-prebuilds
